server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -2518,6 +2518,7 @@ synchronized void finishClearingUnusedLogs() {
   }
 
   private Set<DfsLogger> otherLogs = Collections.emptySet();
+  // logs related to minor compaction file being added to the metadata table
```

Comment:
This wording is a bit confusing. Can you expand on how they are related? Something like "Doomed logs are logs leftover from task X and will be deleted" (or whatever).